lib/flutter_calendar_carousel.dart

```diff
@@ -73,14 +74,14 @@ class CalendarCarousel extends StatefulWidget {
   final Color selectedDayButtonColor;
   final Color selectedDayBorderColor;
   final bool daysHaveCircularBorder;
-  final Function(DateTime) onDayPressed;
+  final Function(DateTime, List<Event>) onDayPressed;
```

Comment:
Could you explain why we need `List<Event>` parameter in `onDayPressed` function?

Reply:
I think this could be better explained in the readme.

Reply:
As we discuss with akuma yesterday, I add it to make it more easier. So you dont need to re select the events and just do foreach on that parameter.